src/Components/Board/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from "react";
+import Cell from "../Cell";
+import "./Board.css";
+
+const Board = () => {
+  const [cellsData, setCellsData] = useState(
+    Array.from({ length: 9 }, () => {
+      return { id: crypto.randomUUID(), value: "" };
+    })
+  );
+  const [isFirstPlayer, setIsFirstPlayer] = useState(true);
+  const [playersScore, setPlayersScore] = useState({ player1: 0, player2: 0 });
+  const [roundCount, setRoundCount] = useState(1);
+  const LINES = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6],
+  ];
+
+  useEffect(() => {
+    for (let i = 0; i < LINES.length; i++) {
+      let indexes = LINES[i];
+      let [index1, index2, index3] = indexes;
+      if (
+        cellsData[index1].value &&
+        cellsData[index2].value &&
+        cellsData[index3].value
+      ) {
+        if (
+          cellsData[index1].value === cellsData[index2].value &&
+          cellsData[index1].value === cellsData[index3].value
+        ) {
+          if (cellsData[index1].value === "X") {
+            setTimeout(() => alert("Winner is Player 1"));
+          } else {
+            setTimeout(() => alert("Winner is Player 2"));
+          }
+          proceedToNextRound();
+        }
+      }
+    }
+  }, [cellsData]);
+
+  const handleClick = (id) => {
+    let updatedCell = cellsData.map((each) => {
+      if (each.id === id) {
+        if (isFirstPlayer) {
+          if (each.value === "") {
+            setIsFirstPlayer(!isFirstPlayer);
+            return { ...each, value: "X" };
+          } else {
+            return { ...each };
+          }
+        } else {
+          if (each.value === "") {
+            setIsFirstPlayer(!isFirstPlayer);
+            return { ...each, value: "O" };
+          } else {
+            return { ...each };
+          }
+        }
+      } else {
+        return each;
+      }
+    });
+    setCellsData(updatedCell);
+  };
+
+  const proceedToNextRound = () => {
+    let boardAfterReset = cellsData.map((cell) => {
+      return { ...cell, value: "" };
+    });
+    setCellsData(boardAfterReset);
+    setRoundCount((previousValue) => {
+      setIsFirstPlayer(previousValue % 2 === 0);
+      return previousValue + 1;
+    });
+    setPlayersScore({ ...playersScore, player1: 0, player2: 0 });
+  };
+
+  const resetBoard = () => {
+    let boardAfterReset = cellsData.map((cell) => {
+      return { ...cell, value: "" };
+    });
```

Comment:
Avoid `return` statements if possible.
```suggestion
    let boardAfterReset = cellsData.map((cell) => ({ ...cell, value: "" }));
```

Reply:
Same everywhere else.